docs/content/en/release-notes/_index.md

```diff
@@ -5,37 +5,52 @@ weight: 2
 description: Release notes for the open-source K8ssandra community project.
 ---
 
-K8ssandra provides a production-ready platform for running Apache Cassandra&reg; on Kubernetes. This includes automation for operational tasks such as repairs, backup and restores, and monitoring. Also deployed is Stargate, an open source data gateway that lets you interact programmatically with your Kubernetes-hosted Cassandra resources via a well-defined API. 
+K8ssandra and K8ssandra Operator provide production-ready platform for running Apache Cassandra&reg; on Kubernetes. This includes automation for operational tasks such as repairs, backup and restores, and monitoring. Also deployed is Stargate, an open source data gateway that lets you interact programmatically with your Kubernetes-hosted Cassandra resources via a well-defined API. 
 
-**Latest release:** K8ssandra 1.4.0
+K8ssandra Operator 1.0.0 is a significant advancement of the overall K8ssandra project. K8ssandra Operator adds:
 
-Release date: 19-November-2021
+* Multi-cluster support for Kubernetes
+* Native support for multi-region/multi-DC Cassandra databases in Kubernetes
+
+As a platform architect or developer, you can declaratively manage the cluster topology and map Cassandra racks to cloud availability zones or failure domains.  
+
+K8ssandra Operator also adds a new observability service, which allows you to choose and deploy your preferred metrics collection and visualization tools. That is, with K8ssandra Operator 1.0.0, you are not limited to using only the Prometheus and Grafana deployed by K8ssandra 1.4.x. 
```

Comment:
So, while this is intended to be true, we haven't actually tried any other tools yet besides prometheus, so we should probably save this for later.

Reply:
Ok, that makes sense. Will comment out or remove, for now.

Reply:
Yeah, I minimized the observability text.